docs/content/docs/integrations/aks.md

```diff
@@ -4,4 +4,147 @@ description: How to connect to an AKS cluster inside a Porter bundle.
+        
+        {
+            "name": "azure_client_secret",
+            "source": {
+                "env": "AZURE_CLIENT_SECRET"
+            }
+        },
+
+        {
+            "name": "azure_subscription_id",
+            "source": {
+                "env": "AZURE_SUBSCRIPTION_ID"
+            }
+        },
+
+        {
+            "name": "azure_tenant_id",
+            "source": {
+                "env": "AZURE_TENANT_ID"
+            }
+        }
+    ]
+}
+```
+
+* Verify that `akscreds` credentials set exists: `porter credentials list`
+
+* Install [az mixin](https://porter.sh/mixins/az/) and declare the credentials in the bundle manifest:
+
+```yaml
+credentials:
+- name: azure_client_id
+  env: AZURE_CLIENT_ID
+  description: AAD Client ID for Azure account authentication - used for AKS Cluster SPN details and for authentication to azure to get KubeConfig
+- name: azure_tenant_id
+  env: AZURE_TENANT_ID
+  description: Azure AAD Tenant Id for Azure account authentication - used to authenticate to Azure to get KubeConfig 
+- name: azure_client_secret
+  env: AZURE_CLIENT_SECRET
+  description: AAD Client Secret for Azure account authentication - used for AKS Cluster SPN details and for authentication to azure to get KubeConfig
+- name: azure_subscription_id
+  env: AZURE_SUBSCRIPTION_ID
+  description: Azure Subscription Id used to set the subscription where the account has access to multiple subscriptions
+```
+
+* Declare Azure resource group and AKS name as parameters:
+```yaml
+parameters:
+  - name: rg_name
+    type: string
+    description: "Resource group in which AKS resides"
+    default: "demorg"
+    applyTo:
+      - merge
+      
+  - name: aks_name
+    type: string
+    description: "AKS name"
+    default: "demoaks"
+    applyTo:
+      - merge
+```
+
+* Bellow you can find a custom action called `merge` that it's used to autheticate in Azure and sets the desired subscription (that contains the resource group and AKS cluster) and uses [outputs](https://porter.sh/wiring/#outputs) to save the *kubeconfig* file:
+
+```bash
+merge:
+  - az:
+      description: "Azure login"
+      arguments:
+        - login
+      flags:
+        service-principal:
+        username: ${ bundle.credentials.azure_client_id }
+        password: ${ bundle.credentials.azure_client_secret }
+        tenant: ${ bundle.credentials.azure_tenant_id }
+  - az:
+      description: "Azure set subscription Id"
+      arguments:
+        - "account"
+        - "set"
+      flags:
+        subscription: ${ bundle.credentials.azure_subscription_id }
+  - az:
+      description: "Get access creds for AKS"
+      arguments:
+        - "aks"
+        - "get-credentials"
+      flags:
+        resource-group: ${ bundle.parameters.rg_name }
+        name: ${ bundle.parameters.aks_name }
+        context: akscluster
+        file: "newconf"
+      suppress-output: false
+      outputs:
+        - name: kubeconfig
+          path: newconf
+
+outputs:
```

Comment:
I would suggest removing the output also to focus on the connection to AKS inside the bundle

Reply:
removed outputs